evennia/VERSION.txt

```diff
@@ -0,0 +1 @@
+0.9
```

Comment:
I guess we need to start discussing just what versioning system scheme to use here.